clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1102,8 +1097,9 @@ public void run() {
   private PushJobHeartbeatSender createPushJobHeartbeatSender(final boolean sslEnabled) {
     try {
       return pushJobHeartbeatSenderFactory.createHeartbeatSender(
+          kafkaTopicInfo.kafkaUrl,
           props,
-          livenessHeartbeatStoreControllerClient,
+          controllerClient,
```

Comment:
The user store and HB store can be in different clusters and hence cannot use the same controller clients in all cases